src/Tests/Backend/Backend.test.ts

```diff
@@ -53,22 +56,39 @@ class ExecutorMockup extends ExecutorBase {
 
 suite('Backend', function() {
   suite('backendRegistrationApi', function() {
+    test('registers a ONEToolchain', function() {
+      let oneBackend = new ONEToolchain();
```

Comment:
```suggestion
      // Register ONE Toolchain backend.
      backendRegistrationApi();
      let oneBackend = new ONEToolchain();
```

Sorry. The `backendRegistrationApi()` function is required to register ONE Toolchain.

Reply:
> Sorry. The `backendRegistrationApi()` function is required to register ONE Toolchain.

Oh, we were about to question that. Because we had discussion and the same conclusion. Thank you for your review! We're going to change the code soon :)

Reply:
Is there anything else to modify? 👀

Reply:
In my humble opinion, there doesn't seem to be anything to modify.